Add more details to fluid-runner readme (#16223)
microsoft/FluidFramework: packages/tools/fluid-runner/README.md

# @fluidframework/fluid-runner

> This package contains utility for running various functionality inside an offline Fluid Framework environment.

Reviewer: I think this still isn't super clear.  I think fluid-runner can be described as "It loads a Fluid container using a provided ODSP snapshot and provided code bundle" or something like that?  I'm not really sure what the "runner" bit is though since it seems like the main focus is on how it loads.

# @fluidframework/fluid-runner

> This package contains utility for running various functionality inside an offline Fluid Framework environment.

Reviewer: ```suggestion
This package contains a tool called "fluid-runner" which can run various functionality inside an offline Fluid Framework environment.
```

> Currently, the only function of the fluid-runner tool is `exportFile`. The tool may be expanded for futher functionality.

Reviewer: "may be expanded for further functionality" -- does this mean we will be adding to it?  Or that we've provided extensibility points for a user to expand on it?  Prob clarify with either:

```suggestion
Currently, the only function of the fluid-runner tool is `exportFile`. We'll be adding more functionality in the future.
```

OR

```suggestion
Currently, the only function of the fluid-runner tool is `exportFile`. However, you can plug in your own functionality by following these steps ________...
```

## Export File

> Loads a Fluid container given a provided ODSP snapshot and code loader bundle, and allows some offline execution to be made on this container. The meaning of "execution" is generalized and it is up to consumer of this tool to determine what it means for their specific scenario.

Reviewer: From the samples below it's not clear to me how the execution is customized - do I need to bake that into the loaded code?  Is there some script I can provide?  Is there some default behavior if I don't customize?
